### DISCUSSION TITLE
"html-button-response" choices size

### DISCUSSION THREAD

**Discussion by GabrielaFacio** [jspsych/jsPsych#1925] [Q&A]
Hello!
I've been trying to create 4 image buttons using the "html-button-response" pluging like this:

```
  var planet1={
  type:"html-button-response",
  stimulus:"",
  choices: ["jup.png", "jup.png", "jup.png", "jup.png"] ,
  button_html:'<button class="jspsych-btn"><img src="%choice%"></button>'};
```

CSS
```
 .jspsych-btnl{
  margin: 0px 8px;
  width: 170px;
  height: 150px;
   display: inline;
 }
```

The problem is that images appear very large on the screen despite having created the corresponding space for each button.
Can anybody help me please? I am new at this.

**Comment by amunn** [comment]
Hi @GabrielaFacio  could it simply be that you have a typo in your CSS or HTML? Your class in the HTML is `jspsych-btn` but your CSS has `.jspsych-btnl`.
